client/components/popover/menu.jsx

```diff
@@ -1,3 +1,4 @@
+/** @format */
```

Comment:
Hmm. Linter is complaining "Missing external, internal dependencies docblocks" even though they're present - related?